x-pack/plugins/fleet/server/services/epm/kibana/index_pattern/install.ts

```diff
@@ -12,7 +12,9 @@ import { appContextService } from '../../..';
 import { getPackageSavedObjects } from '../../packages/get';
 const INDEX_PATTERN_SAVED_OBJECT_TYPE = 'index-pattern';
 
-export const indexPatternTypes = Object.values(dataTypes);
+export const indexPatternTypes = Object.values(dataTypes)
+  // we don't want to add traces-* as a new global data view
+  .filter((dataType) => dataType === dataTypes.Traces);
```

Comment:
```suggestion
  .filter((dataType) => dataType !== dataTypes.Traces);
```

JS's `.filter` method filters down to only elements that pass the check provided, so this should be inverted.

Another option might be to lock this down so that additional data types being added doesn't blow up all the tests again, e.g

```ts
export const indexPatternTypes = [dataTypes.Logs, dataTypes.Metrics]
```

Reply:
Doh, thanks Kyle! I like the allowlist approach for the reason you mentioned — will implement that instead of the filter.

Reply:
Done in 2d3f28b43065d593331f7f958d3cf3cb61ee6e12.